package.json

```diff
@@ -7,7 +7,7 @@
     "test": "mocha"
   },
   "engines": {
-    "node": "4.8.0"
+    "node": "6.3.1"
```

Comment:
We have made a change like to master I believe, we might want to resolve using merging ?

Reply:
I'll take the change from master when I merge the master and dev .